contracts/Vault.vy

```diff
@@ -1582,32 +1584,35 @@ def _assessFees(strategy: address, gain: uint256) -> uint256:
         / SECS_PER_YEAR
     )
 
-    # Only applies in certain conditions
-    strategist_fee: uint256 = 0
-    performance_fee: uint256 = 0
+    self.managementDebt +=  management_fee
+
+    if gain == 0:
+        return 0
```

Comment:
Fee should be at least `management_fee`

Reply:
when gain =0; this time management_fee keep.
it will be deducted next time

Reply:
management_fee can be assets without gains, it's not based on strategy performance but vault managed assets.

Reply:
> when gain =0; this time management_fee keep.
> it will be deducted next time

Please add a test that test managementDebt behavior when gains are 0.